addons/arsenal/functions/fnc_fillRightPanel.sqf

```diff
@@ -272,10 +272,41 @@ switch (_ctrlIDC) do {
         } foreach (GVAR(virtualItems) select 21);
     };
 
+    case IDC_buttonCustom1;
+    case IDC_buttonCustom2;
+    case IDC_buttonCustom3;
+    case IDC_buttonCustom4;
+    case IDC_buttonCustom5;
+    case IDC_buttonCustom6;
+    case IDC_buttonCustom7;
+    case IDC_buttonCustom8;
+    case IDC_buttonCustom9;
+    case IDC_buttonCustom10 : {
+        private _data = GVAR(customRightPanelButtons) param [[RIGHT_PANEL_CUSTOM_BUTTONS] findIf {_x == _ctrlIDC}];
```

Comment:
what is this findIf?
That is just a `find`

Reply:
is findIf not better because not all gets checked?

Reply:
find is not even a loop, so it is infinitely better than findIf, which is a loop.